packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChartWithFallback.base.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/explicit-member-accessibility */
+import * as React from 'react';
+import { IHorizontalBarChartProps } from './HorizontalBarChart.types';
+import { ErrorBoundary } from '../CommonComponents/ErrorBoundary';
+import { HorizontalBarChartBase } from './HorizontalBarChart.base';
+import { isHorizontalBarChartEmpty } from '../CommonComponents/error-utils';
+
+// Create a class to render the HorizontalBarChart within ErrorBoundary
+export class HorizontalBarChartWithFallback extends React.Component<IHorizontalBarChartProps> {
```

Comment:
rename to HorizontalBarChartBaseWithFallback

Reply:
is there any cap on the length of the filenames?

Reply:
lets call it HBCBaseWithFallback etc

Reply:
For all components, I can see we have been following <chartname>.base.tsx so do we need to add a Base to the name as well?

Reply:
Right. We can call it HBCWithFallback.